tools/scripts/release.sh

```diff
@@ -46,7 +46,7 @@ elif [[ ${NEW_SUFFIX} == 'snapshot' ]]; then
   echo new suffix is ${NEW_SUFFIX}
   CURRENT_VERSION=$(jq '.version' ${DIR}/package.json | sed -r "s/\"([0-9]?[0-9]\.[0-9]?[0-9]\.[0-9]?[0-9])(.)*/\1/")
   NEW_VERSION=$( ${DIR}/node_modules/.bin/semver -i ${CURRENT_VERSION} )-snapshot
-  NEW_TAG="unstable-1.4"
+  NEW_TAG="2.0.0-beta.1-1.4"
```

Comment:
Guess this shouldn't still have a 1.4 on the end?

Reply:
that script isn't used anymore